app/helpers/pieces_helper.rb

```diff
@@ -1,2 +1,15 @@
 module PiecesHelper
+  def render_piece_selected(x, y)
+    piece = @game.pieces.find_by(x_position: x, y_position: y, game_id: @game.id)
+    return false unless @selected_piece.present?
+    if @selected_piece.x_position == x && @selected_piece.y_position == y
+      content_tag(:div, 'Selected piece', class: ['selected'])
+    elsif piece = @game.pieces.find_by(x_position: x, y_position: y)
+      link_to "#{piece.type} #{piece.color}", game_path(@game)
+    else
+      link_to "move to", game_path(@game)
```

Comment:
Goal here is to click this link and have it send a request to `pieces#update` with the new x and y coordinates.

A good way to do this is using `button_to`. Docs: http://api.rubyonrails.org/v4.2.7.1/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to

Hints: 
- your path will be `piece_path(@selected_piece)`
- Your `method` needs to be `:put` (this is how Rails know you want to do an update
- You will need to add the new `x_coordinate` and `y_coordinate` in the `button_to`. This will pass the new params to your controller. Then you'll be able to retrieve them using `params` in your controller and use `update_attributes` on the piece.
- ":data - This option can be used to add custom data attributes."